fdbrpc/FlowTransport.h

```diff
@@ -31,7 +31,7 @@
 #include "flow/Net2Packet.h"
 #include "fdbrpc/ContinuousSample.h"
 
-enum { WLTOKEN_ENDPOINT_NOT_FOUND = 0, WLTOKEN_PING_PACKET, WLTOKEN_FIRST_AVAILABLE };
+enum { WLTOKEN_ENDPOINT_NOT_FOUND = 0, WLTOKEN_PING_PACKET, WLTOKEN_AUTH_TENANT, WLTOKEN_FIRST_AVAILABLE };
```

Comment:
If we change WLTOKEN_FIRST_AVAILABLE, maybe we should increase to a higher value like 9, so that we have can add more endpoints here in the future without shifting the existing ones in the fdbclient.